integration-tests/tests/run-workflow.js

```diff
@@ -39,16 +39,13 @@ const testRunWorkflowFn = _.flow(
   await click(page, clickable({ text: 'OK' }))
   await click(page, clickable({ text: 'Run analysis' }))
 
-  // Get request status for sporadically failing checkBucketAccess call.
-  const handler = request => {
-    console.log(`${request.url()} : ${request.status()} : ${request.statusText()}`)
-  }
-  page.on('response', handler)
+  // Uncomment the following to get request status for sporadically failing checkBucketAccess call.
```

Comment:
This comment is confusing to me because the general logging is in effect, so there is no need to uncomment line 43. This is here in case we turn off the general logging?

Reply:
If that is the case, I would suggest

```If general ajax logging is disabled, uncomment the following to debug the sporadically failing checkBucketAccess call.```

Reply:
I was worried about the confusion here. Thanks for the improved wording.